ddtrace/tracer/option_test.go

```diff
@@ -76,6 +78,14 @@ func TestTracerOptionsDefaults(t *testing.T) {
 			assert.Equal(t, c.dogstatsdAddr, "my-host:123")
 		})
 
+		t.Run("env-dd_env", func(t *testing.T) {
```

Comment:
Since the other tests don't contain the env. var's name. Can we not use it here either? Maybe `env-env`. Or, if you really want to, I'd be fine with `env-DD_ENV` too but then use the capitals please as it's more clear.

Reply:
👍 I wasn't sure about this. I only added the dd because `env-env` looked funny to me. If that's ok with you, I'll put it so it's consistent with the others.